lib/registries/addon/components/page-link/component.ts

```diff
@@ -44,7 +44,7 @@ export default class PageLinkComponent extends Component {
             undefined;
     }
 
-    @computed('pageManager', 'pageManager.{isVisited,pageIsValid}', 'pageIsActive')
+    @computed('pageManager', 'pageManager.{isVisited,pageIsValid}', 'pageIsActive', 'metadataIsValid')
```

Comment:
There is an issue when dealing with async validators where the changeset `isValid` remain `true` till the validator returns. This results in the `PageLink` remaining green till the async validator returns, then turns red if there are validation errors. We need take into account `isValidating`, maybe leave the `PageLink` gray or add a new state for this. This is also an existing issue with pages that have `files`.

Reply:
We should make a new ticket for this existing bug.

Reply:
Added a ticket in the notion board